src/Microsoft/Silk.NET.DXGI/DXGILibraryNameContainer.cs

```diff
@@ -0,0 +1,33 @@
+// This file is part of Silk.NET.
+// 
+// You may modify and distribute Silk.NET under the terms
+// of the MIT license. See the LICENSE file for details.
+
+using Silk.NET.Core.Loader;
+
+namespace Silk.NET.DXGI
+{
+    /// <summary>
+    /// Contains the library name of DXGI.
+    /// </summary>
+    internal class DXGILibraryNameContainer : SearchPathContainer
+    {
+        /// <inheritdoc />
+        public override string Linux => "libDXGI.so";
+
+        /// <inheritdoc />
+        public override string MacOS => "libDXGI.dylib";
+
+        /// <inheritdoc />
+        public override string Android => "libDXGI.so";
+
+        /// <inheritdoc />
+        public override string IOS => "__Internal";
```

Comment:
consider adding a comment indicating that this relies on the override.

Reply:
Name containers are not intended for general consumption.

Reply:
I just mean in regards to code readability 😄 
```suggestion
        public override string IOS => "__Internal"; // __Internal relies on a SilkTouch override.
```
am fine if you don't want this too, just figured I'd nit it

Reply:
Committed. It's worth noting that there are tens of other name containers.